massgov/mayflower
# DP-31793 Header alerts

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

- Add a border to the alert header button
- Change the border for alert item container to match the styles to the header button
- Change the icon for notice (yellow one) in alert items 

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-31793)
- [Github issue]()

## Steps to Test


1.  Go to the park location page or header alert component.
2. Find the component has a gray border.
3. Find the border doesn't add up (= get thicker) when it has multiple alert items.
4. The icon for the notice is charged to the bell and it's green.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
